Source/Blazorise/README.md

```diff
@@ -0,0 +1,17 @@
+﻿## Build external libraries
+
+External libraries must be build as a valid JS module to be consumed by Blazorise.
```

Comment:
built*

Do we need to elaborate on the how and what's a JS module in the context of Blazorise?

Reply:
done